sdks/python/apache_beam/examples/wordcount_docker.py

```diff
@@ -0,0 +1,148 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
```

Comment:
Probably extract out these wordcount_xxx updates to a separate PR.

Reply:
Resolving since this was caused due to botched git workflows and have since been corrected.